plugins/woocommerce-admin/client/customize-store/design-with-ai/services.ts

```diff
@@ -220,6 +248,18 @@ export const updateStorePatterns = async (
 			},
 		} );
 
+		const { is_ai_generated } = await apiFetch< {
+			is_ai_generated: boolean;
+		} >( {
+			path: '/wc/private/ai/store-info',
+			method: 'GET',
+		} );
+
+		if ( ! images.length && ! is_ai_generated ) {
```

Comment:
We do the check `is_ai_generated` inside the `resetPatternsAndProducts` function. I think that this is enough:
```suggestion
		if ( ! images.length) {
```

Reply:
You are right! Updating, thanks!! 🙌